client/replays/browse-local-replays.tsx

```diff
@@ -235,23 +242,36 @@ export function ReplayExpansionPanel({ file }: ExpansionPanelProps) {
           <RaceRoot>
             <StyledRaceIcon race={replayRaceToChar(player.race)} />
           </RaceRoot>
-          <PlayerName>{player.isComputer ? 'Computer' : player.name}</PlayerName>
+          <PlayerName>
+            {player.isComputer ? t('game.playerName.computer', 'Computer') : player.name}
+          </PlayerName>
         </PlayerContainer>
       ))
 
       if (teams.size > 1) {
-        elems.unshift(<TeamLabel key={`team-${i}`}>{'Team ' + (i + 1)}</TeamLabel>)
+        elems.unshift(
+          <TeamLabel key={`team-${i}`}>
+            {t('game.teamName.number', {
+              defaultValue: 'Team {{teamNumber}}',
+              teamNumber: i + 1,
+            })}
+          </TeamLabel>,
+        )
       }
 
       return elems
     })
 
     return [durationStr, gameTypeLabel, mapName, playerListItems]
-  }, [mapInfo?.name, replayMetadata, replayUserIds, usersById])
+  }, [mapInfo?.name, replayMetadata?.headerData, replayUserIds, t, usersById])
 
   let content
   if (parseError) {
-    content = <ErrorText>There was an error parsing the replay</ErrorText>
+    content = (
+      <ErrorText>
+        {t('replays.local.parseError', 'There was an error parsing the replay')}
```

Comment:
I don't think "parse" is going to be a familiar term to users, we should probably just change this to "There was a problem loading the replay" or "There was a problem processing the replay"